packages/@react-spectrum/menu/test/MenuTrigger.test.js

```diff
@@ -778,4 +779,133 @@ describe('MenuTrigger', function () {
     let menu2Item1 = within(menu2).getByText('Whiskey');
     expect(menu2Item1).toBeInTheDocument();
   });
+
+  describe('MenuTrigger trigger="longPress" open behavior', function () {
+    installPointerEvent();
+
+    const DEFAULT_THRESHOLD = 500;
+
+    it('should open the menu on longPress', function () {
+      const props = {onOpenChange, trigger: 'longPress'};
+      verifyMenuToggle(MenuTrigger, props, {}, (button, menu) => {
+        if (!menu) {
+          triggerLongPress(button);
+        } else {
+          triggerTouch(button);
+        }
+      });
+    });
+
+    it('should open the menu on Enter', function () {
+      const props = {onOpenChange, trigger: 'longPress'};
+      verifyMenuToggle(MenuTrigger, props, {}, (button, menu) => {
+        if (!menu) {
+          fireEvent.keyDown(button, {key: 'Enter'});
+          act(() => jest.advanceTimersByTime(DEFAULT_THRESHOLD));
+          fireEvent.keyUp(button, {key: 'Enter'});
```

Comment:
I'm reading this test as a long `enter` key press and when I test that in storybook instantly opens. I'm either missing what this test is testing or it doesn't really test a long `enter` key press.

Reply:
thanks for pointing this out! yes it doesn't really test a long enter key press. The expected behavior should be 'short press' enter to open the menu. I'll fix this in incoming commits.